<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>861</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="hrzZoom">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>871</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>450</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRenderingRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rendering rate:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>10</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Task Selection</string>
    </property>
    <widget class="QGroupBox" name="groupBoxOpenGL">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>211</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>OpenGL Options</string>
     </property>
     <widget class="QRadioButton" name="optGLBufferData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Buffer Data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="optGLMapBuffer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>54</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Map Buffer</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblGLBufferData">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>27</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblGLMapBuffer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>54</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxIO">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>31</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>I/O Options</string>
     </property>
     <widget class="QRadioButton" name="optSTL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>STL</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="optMMF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>54</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MMF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSTL">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblMMF">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxOpenCL">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>121</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>OpenCL Options</string>
     </property>
     <widget class="QRadioButton" name="optCLDefault">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="optCLInteroperability">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Interoperability</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCLInteroperability">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>52</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblCLDefault">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>25</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
